context/app/static/js/components/detailPage/BulkDataTransfer/usePanelSet.jsx

```diff
@@ -180,31 +184,39 @@ const NON_CONSORTIUM_MEMBERS_NO_DBGAP = {
 };
 
 const DATASET_NOT_FINALIZED = {
-  error: (
-    <div>
-      These data are still being prepared, processed, or curated and will only be available to members of the team who
-      submitted the data. For additional help, email
-      <EmailIconLink variant="body2" email="help@hubmapconsortium.org">
-        help@hubmapconsortium.org
-      </EmailIconLink>
-      .
-    </div>
-  ),
+  error: {
+    status: 'warning',
+    children: (
+      <div>
+        These data are still being prepared, processed, or curated and will only be available to members of the team who
+        submitted the data. For additional help, email
+        <EmailIconLink variant="body2" email="help@hubmapconsortium.org">
+          help@hubmapconsortium.org
+        </EmailIconLink>
+        .
+      </div>
+    ),
+  },
 };
 
 const ENTITY_API_ERROR = {
-  error: (
-    <div>
-      The API failed to retrieve the link to Globus. Please report this issue to&nbsp;
-      <EmailIconLink variant="body2" email="help@hubmapconsortium.org">
-        help@hubmapconsortium.org
-      </EmailIconLink>
-      &nbsp;with the dataset ID and information about the files you are trying to access.
-    </div>
-  ),
+  error: {
+    status: 'error',
+    title: 'HuBMAP Globus Access',
+    tooltip: globusText.tooltip,
+    children: (
+      <>
```

Comment:
Out of curiosity - I noticed the `warning` panels are wrapped in a `div`, while this `error` panel is using a fragment. Is this related to the toggle between `block` and `flex`?

Reply:
There is no particular reason for this, some of them are divs and some of them are fragments. 

If this is something we want to be particular on, then we would need to decide on which one we want to go with, as they are pretty much being used interchangeably.